corehq/apps/app_manager/dbaccessors.py

```diff
@@ -324,3 +324,8 @@ def get_case_types_from_apps(domain):
          .size(0)
          .terms_aggregation('modules.case_type.exact', 'case_types'))
     return set(q.run().aggregations.case_types.keys)
+
+
+def get_case_sharing_apps_in_domain(domain, exclude_app_id=None):
```

Comment:
This was already being called. I just moved it to dbaccessors https://github.com/dimagi/commcare-hq/pull/12826/files#diff-8a78f7edae07ff768accdb85fbaace47L214

Can't use `get_brief_apps_in_domain` as `ParentCasePropertyBuilder` looks through modules and forms structures